include/aspect/utilities.h

```diff
@@ -164,18 +169,19 @@ namespace aspect
       /**
        * This enum lists available coordinate systems that can be used for
        * the function variables. Allowed values are 'cartesian',
-       * 'spherical', and 'depth'. 'spherical' coordinates follow: r, phi
-       * (2D) or r, phi, theta (3D); where r is radius, phi is longitude,
-       * and theta is the polar angle (colatitude). The 'depth' is a
-       * one-dimensional coordinate system in which only the distance
-       * below the 'top' surface (depth) as defined by each geometry model,
+       * 'spherical', 'ellipsoidal', and 'depth'. 'spherical' and 'ellipsoidal'
+       * coordinates follow: r, phi (2D) or r, phi, theta (3D); where r
+       * is radius, phi is longitude, and theta is the polar angle (colatitude).
+       * The 'depth' is a one-dimensional coordinate system in which only the
+       * distance below the 'top' surface (depth) as defined by each geometry model,
        * is used.
        */
-      enum CoordinateSystem
+      enum CoordinateSystem : unsigned short
```

Comment:
is it important for anything in this patch to declare that the width of the `enum` is `unsigned short`? If not, just let the compiler figure out the appropriate integer type.

Reply:
Yes, because it is not possible to forward declare an enum (which I currently need ininclude/aspect/geometry_model/interface.h) , except if you septically assign a type to it.

Reply:
But this is a C++11 feature to derive an enum from an integer type. I don't think you can use that yet.

Reply:
hmm, you are right. That is slightly problematic, because what I read is that forward declarations of enums are not possible before c++11. So I will have to rethink this solution.